### PR TITLE
Add script to get base checksum of an image in dockerhub

### PR DESCRIPTION
**Pull request by aandvalenzuela**

### CODE REVIEW

File: bin/get_base_checksum.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from docker_utils import get_labels
+
+parser = ArgumentParser()
+parser.add_argument(
+    "-n", "--name", dest="name", help="Docker image name e.g cmssw/el8:aarch64-runtime", type=str
+)
+args = parser.parse_args()
+
+labels = get_labels(args.name)
+print(labels['build-checksum'])
```
Comment (smuzaffar) [inline bin/get_base_checksum.py:12]: @aandvalenzuela , this only works for our images and will not work for baseimage like `almalinux:8` .... right?

Comment (aandvalenzuela) [reply bin/get_base_checksum.py:12]: Yes, only for the images in our dockerhub

Comment (smuzaffar) [reply bin/get_base_checksum.py:12]: where are you going to use it? build container job should already know this checksum

Comment (aandvalenzuela) [reply bin/get_base_checksum.py:12]: For example, when building `grid` images, the grid build container know its own checksum, but I need to get the checksum of the base `runtime`.

**Comment by aandvalenzuela** [comment]
Closing in favor of keeping track of the almalinux base image.

**Pull request closed without merging**